### ISSUE TITLE
StartTransaction cannot determine if transactions are supported because there are no connected servers` when trying to start the transaction on `ReplSet` mod

### ISSUE THREAD

**Issue by ivanparvaresh** [Mongo2Go/Mongo2Go#91]
# Summary 
`MongoDbRunner.Start` on `singleNodeReplSet:true` mode throws 
`StartTransaction cannot determine if transactions are supported because there are no connected servers` when trying to start the transaction.

But when I'm running this test on the Mongo2Go project it passes.  

# Steps:
```c#
var runner= MongoDbRunner.Start(singleNodeReplSet: true);
var client = new MongoClient(this.runner.ConnectionString);
var database = client.GetDatabase("testdb");
var session = client.StartSession();
session.StartTransaction(); <--------- The Exception part
```

## Exception
```
System.NotSupportedException : StartTransaction cannot determine if transactions are supported because there are no connected servers.
  Stack Trace:
     at MongoDB.Driver.Core.Bindings.CoreSession.EnsureTransactionsAreSupported()
   at MongoDB.Driver.Core.Bindings.CoreSession.EnsureStartTransactionCanBeCalled()
   at MongoDB.Driver.Core.Bindings.CoreSession.StartTransaction(TransactionOptions transactionOptions)
   at MongoDB.Driver.ClientSessionHandle.StartTransaction(TransactionOptions transactionOptions)
```

# Environment
.NET Core SDK (reflecting any global.json):
 Version:   3.1.100
 Commit:    cd82f021f4

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.15
 OS Platform: Darwin
 RID:         osx.10.15-x64
 Base Path:   /usr/local/share/dotnet/sdk/3.1.100/

Host (useful for support):
  Version: 3.1.0
  Commit:  65f04fb6db

.NET Core SDKs installed:
  3.0.100 [/usr/local/share/dotnet/sdk]
  3.1.100 [/usr/local/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.App 3.0.0 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.0 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.13 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.0.0 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.0 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download

Also, I'm using `xunit` as my TestFramework.

**Comment by AndreasTruetschel** [comment]
I think this is a dupe of #89  
I have a workaround described there.

**Comment by Roger-Meng** [comment]
The issue remains, even if I tried to ensure all collections created before StartTransaction().

```
        public async Task EnsureCollection()
        {
            await Database.EnsureCollectionAsync("customers");
            await Database.EnsureCollectionAsync("sessions");
            await Database.EnsureCollectionAsync("suggestions");
            await Database.EnsureCollectionAsync("idreqs");
        }
```

**Comment by maint16** [comment]
I had the same issue. Is there any solution?

**Comment by zegsampaio** [comment]
Same here. Any perspective of the issue resolved? Thank you

**Comment by JohannesHoppe** [comment]
I have no resources for active development. But anybody here is welcome to create a PR!

**Comment by JohannesHoppe** [comment]
Looks like it's the same issue as i #100 which is may be fixed by #101 . I will close this one to concentrate on #100.

**Comment by JohannesHoppe** [comment]
New version released. Please try out Mongo2Go 2.2.13.
